api-client/projects/vcd/schematics/src/schematics/plugin-seed/index.d.ts

```diff
@@ -1,3 +1,17 @@
-import { Rule } from '@angular-devkit/schematics';
```

Comment:
I think the plugin-seed updates won't be needed anymore. It won't hurt, but probably not worth investing time into it.